ui/hooks/useUserPreferencedCurrency.js

```diff
@@ -8,6 +8,7 @@ import { getNativeCurrency } from '../ducks/metamask/metamask';
 
 import { PRIMARY, SECONDARY } from '../helpers/constants/common';
 import { EtherDenomination } from '../../shared/constants/common';
+import { ETH_DEFAULT_DECIMALS } from '../pages/confirmations/constants';
```

Comment:
I was wondering about how we go about importing logic from pages/confirmations in outside files. Should we utilize the helpers or shared directories for these?